frontend/resources/translations/en.json

```diff
@@ -615,7 +615,6 @@
       "freeLabels": {
         "itemOne": "charged per workspace",
         "itemTwo": "3",
-        "itemThree": "",
```

Comment:
@suntorytimed The test failed because you deleted this translation.

Reply:
Interesting, I didn't actively touch them. I will readd them, but this seems to be something inlang did.